package.json

```diff
@@ -68,6 +68,7 @@
     "@storybook/addon-interactions": "^7.0.8",
     "@storybook/addon-links": "^7.0.8",
     "@storybook/blocks": "^7.0.8",
+    "@storybook/jest": "^0.1.0",
```

Comment:
How often is this module maintained?

I'm worried that we're needlessly more and more adding packages to our website. I think, for now, it's okay, but let's keep this in mind. We don't want to have too many packages accidentally by the end of the day...

Let's keep this on mind!

Reply:
This module seems to be actively maintained. The last version about a month old.